src/index.html

```diff
@@ -10,8 +10,197 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="field-group">
+        <legend>Personal information</legend>
+        <div class="form-controls">
+          <label for="surname">
+            Surname:
+          </label>
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            minlength="2"
+            maxlength="30"
+            id="surname"
+            required
+          >
+        </div>
```

Comment:
@MabuzaM you could optimize your code drastically by nesting `<input`s inside `<label`s like so:
```suggestion
          <label for="surname" class="form-controls">
            Surname:
            <input
              type="text"
              autocomplete="off"
              name="surname"
              minlength="2"
              maxlength="30"
              id="surname"
              required
            >
          </label>
```